FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -30,7 +30,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if !UserDefaults.standard.bool(forKey: UserDefaultsKeys.hasShownWelcomeScreen.rawValue) {
             showWelcome()
         }
-        
+
+        var types = UIUserNotificationType()
+        types.insert(.alert)
+        types.insert(.badge)
+        UIApplication.shared.registerUserNotificationSettings(UIUserNotificationSettings(types: types, categories: nil))
```

Comment:
`UserNotifications` is the new framework, the old stuff was actually part of UIKit. 

I have some [sample code supporting both 9 and 10](https://github.com/designatednerd/iOS10NotificationSample/) you can yoink if you want.